src/test/java/dev/openfeature/javasdk/ProviderSpecTest.java

```diff
@@ -45,7 +45,7 @@ public class ProviderSpecTest {
             "reason field with a string indicating the semantic reason for the returned flag value.")
```

Comment:
The 2.6 spec fails the `spec_finder.py` check. I think because it was updated with a list of common reasons.